src/components/gltf-model-plus.js

```diff
@@ -58,7 +58,8 @@ const extractZipFile = promisifyWorker(new SketchfabZipWorker());
 
 function defaultInflator(el, componentName, componentData) {
   if (!AFRAME.components[componentName]) {
-    throw new Error(`Inflator failed. "${componentName}" component does not exist.`);
+    console.warn(`Inflator failed. "${componentName}" component does not exist.`);
```

Comment:
We intentionally don't load all components on the scene page, but this causes some models to fail to load. This doesn't seem like it should be fatal, so converted it to a warning.